Objects/moduleobject.c

```diff
@@ -162,6 +162,63 @@ _add_methods_to_object(PyObject *module, PyObject *name, PyMethodDef *functions)
     return 0;
 }
 
+int
+PyModule_AddConstants(PyObject *module, PyModuleConst_Def *def)
+{
+    PyObject *dict;
+    PyModuleConst_Def *cur_def;
+    PyObject *v;
+    int res;
+
+    dict = PyModule_GetDict(module);
+    if (dict == NULL) {
```

Comment:
There have an `assert()` in `PyModule_GetDict()`, so `dict == NULL` can be removed?

REF: https://github.com/python/cpython/blob/master/Objects/moduleobject.c#L465